parser/class_files.py

```diff
@@ -86,31 +86,30 @@ def parse_images(tag, target, dita_soup):
     #Find the colspan:6 table
     td = tag.find("td", {"colspan": "6"})
 
-    #Find the parent table (div id="Table")
-    if td is not None:
+    if td:
+        #Find the parent table (div id="Table")
         parent_table = td.find_parent("div", id="Table")
 
         #Find the parent element of the table
-        page_layer = parent_table.find_parent("div", id="PageLayer")
+        parent_div = parent_table.parent("div")
 
-        for child in page_layer.children:
-            #Find the img child
-            img = child.find("img")
-            if  type(child) is bs4.element.Tag and img is not None:
+        for div in parent_div:
+            img = div.find("img")
```

Comment:
I know this works for the current data, but we need to allow for multiple `img` in the `div`.  So, this needs to be a `find_all` -> and loop through matches.

I'll modify the data in this branch to introduce an example.